### PR TITLE
Shop

### PR DESCRIPTION
**Pull request by khush2003**
Have done the basic pages, added product display component and ShopAppBody

### CODE REVIEW

File: frontend/src/components/shop/ProductDisplay.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+const ProductDisplay: FC<{
+    name: string
+    image: string
+    brandName: string
+    price: string
+    link: string
+}> = ({ name, image, brandName, price, link }) => {
+    return (
+        <div>
+            <Box pt-6 background="white" width="223px" height="261px" borderWidth="1px" borderRadius="lg" overflow="hidden">
```
Comment (ThisTine) [inline frontend/src/components/shop/ProductDisplay.tsx:13]: What is width 233px and height 261px, is this going the be responsive on mobile right ?

**Review by ThisTine — CHANGES_REQUESTED** [review]
Please check on width and height and make sure that they will be responsive on all devices.
or if you plan to do this on this sprint (Sprint 1) please tell me so I can just merge and check it again on the next pull request.

File: frontend/src/components/shop/ProductDisplay.tsx (hunk 1 above)
Comment (khush2003) [reply frontend/src/components/shop/ProductDisplay.tsx:13]: Yes, we will make it responsive in next Sprint

Comment (khush2003) [reply frontend/src/components/shop/ProductDisplay.tsx:13]: The width and height for now are only for making the component initially

Comment (ThisTine) [reply frontend/src/components/shop/ProductDisplay.tsx:13]: Ok then there is no problem here.

**Review by ThisTine — APPROVED** [review]

**Pull request merged**